sunnypilot/selfdrive/car/interfaces.py

```diff
@@ -0,0 +1,37 @@
+"""
+Copyright (c) 2021-, Haibin Wen, sunnypilot, and a number of other contributors.
+
+This file is part of sunnypilot and is licensed under the MIT License.
+See the LICENSE.md file in the root directory for more details.
+"""
+
+from opendbc.car import Bus, structs
+from opendbc.car.can_definitions import CanRecvCallable, CanSendCallable
+from opendbc.car.car_helpers import can_fingerprint
+from opendbc.car.hyundai.radar_interface import RADAR_START_ADDR
+from opendbc.car.hyundai.values import HyundaiFlags, DBC as HYUNDAI_DBC
+from opendbc.sunnypilot.car.hyundai.values import HyundaiFlagsSP
+
+
+def setup_car_interface_sp(CP: structs.CarParams, params):
+  if CP.carName == 'hyundai':
```

Comment:
I am a little hesitant on having it this way here. I feel there's somewhere else we can do were this is coded for the interface instead of us having to have here a list of potentially multiple makes and bunch of ifs

Reply:
We probably should do it closer to what we do for MADS where we segregate by brands